man/gbz80.7

```diff
@@ -43,23 +43,36 @@ Any of the 8-bit registers
 Any of the general-purpose 16-bit registers
 .Pq Sy BC , DE , HL .
 .It Ar n8
-8-bit integer constant.
+8-bit integer constant
+.Po signed or unsigned,
```

Comment:
Can't we tell from the carry flag that n8 and n16 are actually unsigned?

Example:
```
LD A, 1
ADD A, $ff
```
I assume this sets the C flag which it wouldn't if $ff was signed.

Reply:
This is about assembly time, not runtime: "signed or unsigned" here means the allowed range for the assembler is -$80 to $FF, not just unsigned $00 to $FF or just signed -$80 to $7F (so `ld a, -128` and `ld a, 255` are both valid for RGBASM to assemble).